packages/sysutils/udevil/config/udevil.conf

```diff
@@ -230,6 +230,7 @@ default_options_ftpfs     = nosuid, noexec, nodev, noatime, uid=$UID, gid=$GID
 default_options_davfs     = nosuid, noexec, nodev, uid=$UID, gid=$GID
 default_options_tmpfs     = nosuid, noexec, nodev, noatime, uid=$UID, gid=$GID
 default_options_ramfs     = nosuid, noexec, nodev, noatime, uid=$UID, gid=$GID
+default_options_btrfs     = compress=zstd:3
```

Comment:
I imagine games are mostly read, so lz4 being the cheapest decompressor would be the best fit for these underpowered CPUs.